packages/elements-react/src/components/index.ts

```diff
@@ -1,216 +1,61 @@
-import { JSX as StencilJSX } from '@stencil/core';
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore
-import { addIcons } from '@inovex.de/elements/dist/collection/util/icons';
-import { ICON_PATHS } from '@inovex.de/elements/dist/inovex-elements/ino-icon/index.esm.js';
-import { defineCustomElements } from '@inovex.de/elements/dist/loader';
-
-import { createReactComponent } from './createComponent';
+/* eslint-disable */
+/* tslint:disable */
+/* auto-generated react proxies */
+import { createReactComponent } from './react-component-lib';
 
-defineCustomElements(window);
-addIcons(ICON_PATHS);
```

Comment:
I suppose that this file is auto-generated. Is there a way to keep the `addIcons(...)` function?